### DISCUSSION TITLE
Remove K32 plots as K34 is ready to be moved

### DISCUSSION THREAD

**Discussion by FVierra27** [ericaltendorf/plotman#958] [Ideas]
Thinking ahead here. I haven't plotted in many month at this point but it would be nice to plot a K34 and when it's ready delete some K32s and replace them.

**Comment by altendky** [comment]
This is an often requested item but I have been fairly hesitant to add deletion of plots to plotman.  Any bug around that could be really bad.  When I was replotting from OG to pooling plots I just deleted either a days worth of plots at a time or a disk worth.  I know some other people did put together scripts for this and there are other discussions and issues around this, though I wasn't able to quickly find them.

**Comment by zorner** [comment]
#963 I just posted my approach.
